charts/csm-drivers-modules/Chart.yaml

```diff
@@ -0,0 +1,75 @@
+#
+#
+# Copyright © 2023 Dell Inc. or its subsidiaries. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#      http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+#
+apiVersion: v2
+name: csm-drivers-modules
```

Comment:
Can the name of the chart be "csm" and the folder location changed to "charts/csm". This may make it easier to understand as a single helm chart for CSM and we don't need to mention "drivers-modules" in the name.

Reply:
Thanks, Trevor, for reviewing..I renamed the chart as suggested